### ISSUE TITLE
Add --log-level command-line argument

### ISSUE THREAD

**Issue by Nuru** [cloudposse/atmos#370]
### Describe the Feature

#359 added Log Level control via `atmos.yaml` and the `ATMOS_LOGS_LEVEL` ENV var. For consistency with other options and other CLI tools, please add `--log-level` as a global command-line argument. 

### Expected Behavior

```
$ atmos --log-level Trace  workflow vendor -f identity
Error: unknown flag: --log-level
```

### Use Case

Override default log level on a single command basis.

### Describe Ideal Solution

Add --log-level command-line argument

### Alternatives Considered

Yes, you can override like this:

```
$ ATMOS_LOGS_LEVEL=Trace atmos workflow vendor -f identity
```

but that is clunky. 

### Additional Context

The idea of setting a logging level is widespread, and is almost universally referred to as `LOG_LEVEL`. That is why I suggest the command-line argument `--log-level` rather than `--logs-level`. I would also suggest you rename the ENV VAR `ATMOS_LOG_LEVEL` instead of `ATMOS_LOGS_LEVEL`. I understand you have a convention that the environment variable name is parallel to the `atmos.yaml` configuration name, but I would prefer you change the configuration key from `logs` to `log` rather than break this widespread convention.

**Comment by GabisCampana** [comment]
https://cloudposse.atlassian.net/browse/DEV-1675

**Comment by osterman** [comment]
This is already implemented in #532
